backend/src/Designer/Controllers/ResourceAdminController.cs

```diff
@@ -78,9 +79,43 @@ public ActionResult<ServiceResource> GetResourceById(string org, string reposito
         [HttpGet]
         [Route("designer/api/{org}/resources/repository/validate/{repository}")]
```

Comment:
I belvie we should remove the repository in path. I think it is confusing.

Reply:
Oh yes I agree. I think I'll remove it from all the routes in the controller as a part of this PR as well.